ovs/lib/tc.c

```diff
@@ -2673,7 +2673,7 @@ nl_msg_put_flower_options(struct ofpbuf *request, struct tc_flower *flower)
         FLOWER_PUT_MASKED_VALUE(ip_ttl, TCA_FLOWER_KEY_IP_TTL);
         FLOWER_PUT_MASKED_VALUE(ip_tos, TCA_FLOWER_KEY_IP_TOS);
 
-        if (flower->mask.ip_proto && flower->key.ip_proto) {
+        if (flower->mask.ip_proto || flower->key.ip_proto) {
```

Comment:
I think this should be && only

Reply:
No this should be || only, either of them is true, we have to add it in netlink msg